Bottom: d94f1e9e819297d97bd979b9c336e3199db760b3
Top:    662a846c8120f9bd9b0ff20f430fa9ca3781869e
Author: Ian Kent <ikent@redhat.com>
Date:   2013-09-23 14:57:40 +0800

Refresh of debug.patch

---

diff --git a/lib/rpc_subs.c b/lib/rpc_subs.c
index d1f8b82..a3444c4 100644
--- a/lib/rpc_subs.c
+++ b/lib/rpc_subs.c
@@ -588,7 +588,7 @@ int rpc_portmap_getclient(struct conn_info *info,
 	info->addr = addr;
 	info->addr_len = addr_len;
 	info->program = rpc_getrpcbyname(rpcb_prog);
-	info->port = rpc_getrpcbport(proto);
+	info->port = ntohs(rpc_getrpcbport(proto));
 	info->version = rpcb_version;
 	error(LOGOPT_ANY, "program %d port %d version %d", info->program, info->port, info->version);
 	info->proto = proto;
@@ -633,7 +633,7 @@ int rpc_portmap_getport(struct conn_info *info,
 		pmap_info.host = info->host;
 		pmap_info.addr = info->addr;
 		pmap_info.addr_len = info->addr_len;
-		pmap_info.port = rpc_getrpcbport(info->proto);
+		pmap_info.port = ntohs(rpc_getrpcbport(info->proto));
 		pmap_info.program = rpc_getrpcbyname(rpcb_prog);
 		pmap_info.version = rpcb_version;
 		error(LOGOPT_ANY, "program %d port %d version %d", pmap_info.program, pmap_info.port, pmap_info.version);
